Gopkg.toml

```diff
@@ -5,22 +5,10 @@
   name = "github.com/ghodss/yaml"
   version = "1.0.0"
 
-[[constraint]]
```

Comment:
I am curious: Do you remove these packages automatically somehow or is this a manual job?

Reply:
Manually for Gopkg.toml, had to go through to see what was being used by the CLI. `dep` will pick up dependencies from the imports in files most of the time, but this is needed when you need to lock to a very specific version/commit (which for a lot of these we had to do to ensure everything aligned with the right client-go version).